services/frontend-service/src/ui/Pages/Home/Home.test.tsx

```diff
@@ -32,20 +32,278 @@ describe('App', () => {
     const getWrapper = () => render(getNode());
     it('Renders full app', () => {
         // when
+        const appBuilder = (suffix: string) => ({
```

Comment:
```suggestion
        const buildApp = (suffix: string) => ({
```
Would make it sound more like a function

Reply:
renamed to `buildTestApp` to also be more explicit about the fact that it only builds simple test applications